nidx/nidx_protos/build.rs

```diff
@@ -22,13 +22,15 @@ use std::io::Result;
 
 fn main() -> Result<()> {
     println!("cargo:rerun-if-changed=../../nucliadb_protos");
+    println!("cargo:rerun-if-changed=src");
     tonic_build::configure().emit_rerun_if_changed(false).compile_protos(
         &[
+            "src//nidx.proto",
```

Comment:
Double slash for any reason?
```suggestion
            "src/nidx.proto",
```

Reply:
Oops :upside_down_face: